ddtrace/llmobs/_integrations/utils.py

```diff
@@ -135,9 +135,15 @@ def get_system_instructions_from_google_model(model_instance):
     Extract system instructions from model and convert to []str for tagging.
     """
     from vertexai.generative_models._generative_models import Part
-
+    from google.ai.generativelanguage_v1beta.types.content import Content
```

Comment:
Just to be safe - can we put these imports in a try/catch just in case a vertexAI user does not have google_generativeai installed/available and vice versa?